src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -216,9 +216,13 @@ export const TradeAmountInput: React.FC<TradeAmountInputProps> = memo(
 
     const oppositeCurrency = useMemo(() => {
       return isFiat ? (
-        <Amount.Crypto value={cryptoAmount ?? ''} symbol={assetSymbol} prefix='≈' />
+        <Amount.Crypto
+          value={bnOrZero(cryptoAmount).gt(0) ? cryptoAmount : '0'}
+          symbol={assetSymbol}
+          prefix='≈'
+        />
       ) : (
-        <Amount.Fiat value={fiatAmount ?? ''} prefix='≈' />
+        <Amount.Fiat value={bnOrZero(fiatAmount).gt(0) ? fiatAmount : '0'} prefix='≈' />
```

Comment:
nitpick: since we already `bnOrZero(fiatAmount)` we could just use it as a value

Reply:
gm mister big 🧠  , fixed in [28f11b5](https://github.com/shapeshift/web/pull/7297/commits/28f11b5aee3dc4051678b362bddd58d94de4a318)